azurerm/internal/services/bot/bot_channels_registration_resource.go

```diff
@@ -124,6 +138,18 @@ func resourceBotChannelsRegistration() *pluginsdk.Resource {
 				ValidateFunc: validation.IsUUID,
 			},
 
+			"icon_url": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: validate.BotChannelRegistrationIconUrl,
+			},
+
+			"is_isolated_network_enabled": {
```

Comment:
we can probably remove is?
```suggestion
			"isolated_network_enabled": {
```

Reply:
Updated